lib/sequenceserver/blast.rb

```diff
@@ -104,6 +103,10 @@ def pre_process(params)
         params[:sequence].strip! unless params[:sequence].nil?
       end
 
+      def blast_formatter(rfile_path, ofmt, specifiers = nil)
+        `blast_formatter -archive '#{rfile_path}' -outfmt '#{ofmt} #{specifiers}'`
```

Comment:
`specifiers` is technically a user input. Thus should be verified, and used only if applicable. That is, only for TSV, and only after checking its validity.